javascript/javascript2/week1/shortword.js

```diff
@@ -0,0 +1,11 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"]; // returns 'ø'
+function findShortest(d) {
```

Comment:
Find a better name for d. like bellow
```suggestion
function findShortest(words) {
```